openquake/hazardlib/gsim/bradley_2013.py

```diff
@@ -47,7 +48,11 @@
 
 
 def _get_basin_term(C, ctx, region=None):
-    z1pt0 = ctx.z1pt0
+    z1pt0 = copy.deepcopy(ctx.z1pt0)
```

Comment:
In 100 places you do this `copy.deepcopy` of an array, but the idiomatic way is to write `ctx.z1pt0.copy()`, directly using the numpy functionality. `copy.deepcopy` is meant to be used on nested structures, not on simple arrays.